composer.json

```diff
@@ -37,5 +38,14 @@
 		"branch-alias": {
 			"dev-master": "1.x-dev"
 		}
+	},
+	"repositories": {
```

Comment:
composer/installers was added as a dependency into composer.json because we found that otherwise the extension would not work when installed via composer.